CHANGELOG.md

```diff
@@ -3,11 +3,23 @@
 Observes [Semantic Versioning](https://semver.org/spec/v2.0.0.html) standard and 
 [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) convention.
 
+## 0.1.0 - 2022-06-29
+
++ Add - Suite2p triggering pytest
++ Update - Docker and Compose files
++ Add - GitHub issue templates
++ Add - Paired jupytext python scripts
++ Add - Analysis schema for event-aligned calcium activity
++ Add - Citation section to README
++ Add - YouTube tutorial link to README
++ Update - Requirements
++ Add - Adopt black formatting into code base
+
 ## 0.1.0a2 - 2022-01-24
-### Updated
-+ Changed version to reflect release phase
+
++ Update - Change version to reflect release phase
 
 ## 0.1.1 - 2021-03-26
```

Comment:
Is 0.1.1 an error from 2021? 0.1.1 should have bumped to 0.1.1a2, right? That would make our current version 0.1.2, right?